tests/test_requests.py

```diff
@@ -1188,6 +1188,17 @@ def test_response_iter_lines_reentrant(self, httpbin):
         next(r.iter_lines())
         assert len(list(r.iter_lines())) == 3
 
+    def test_session_close_proxy_clear(self, mocker):
+        proxies = {
+          'one': mocker.Mock(autospec=True),
```

Comment:
Why are you using `autospec` here? You're not actually spec'ing out a ProxyManager.